PODMAN.md

```diff
@@ -57,3 +57,18 @@ and `KUBEVIRTCI_PODMAN_SOCKET` accordingly,
 i.e `export KUBEVIRTCI_PODMAN_SOCKET="${XDG_RUNTIME_DIR}/podman/podman.sock"`
 
 Tested on fedora 35.
+
+## Resource Adjustments
+
+When working with Podman, you might encounter PID resource constraints. To resolve this issue:
+
+1. Locate and edit your `containers.conf` file (typically in `/usr/share/containers`)
+2. Add or modify the PID limit setting:
+    ```toml
+    [containers]
+    # Configure the process ID (PID) limit for containers
+    # Options:
+    #   Numeric value: Sets specific PID limit (e.g., 2048)
+    #   -1: Removes PID limitations entirely
```

Comment:
Maybe we should suggest a value to the user that works for most of us. @stu-gott what is the value you have experienced to work for you?